src/Serilog/Core/ForContextExtension.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using Serilog.Events;
+
+namespace Serilog
+{
+    /// <summary>
+    /// Extension method 'ForContext' for ILogger.
+    /// </summary>
+    public static class ForContextExtension
+    {
+        /// <summary>
+        /// Create a logger that enriches log events with the specified property based on log event level.
```

Comment:
`Create a logger that enriches log events when the specified level is enabled.` ?